src/components/EventCalender/Weekly/WeeklyEventCalendar.spec.tsx

```
@@ -0,0 +1,179 @@
+import React, { Suspense } from 'react';
+import {
+  render,
+  fireEvent,
+  act,
+  screen,
```

ESLint

[error] 'screen' is defined but never used.

(@typescript-eslint/no-unused-vars)